src/plugins/GetMetadataPlugin.ts

```diff
@@ -29,6 +29,7 @@ type MetaEntry = { key: string; value: MetaType };
 
 export function CreateMetadataPlugin(schemaName: string) {
   return makeExtendSchemaPlugin((build) => {
+    // TODO Only handled the single-chain scenario, multi-chains may have unexpected results.
```

Comment:
Whats the current behaviour? I think its probably fine we dont support it but also we should provide an appropriate error if we don't